### ISSUE TITLE
Markdown embed include photos

### ISSUE THREAD

**Issue by zsviczian** [zsviczian/obsidian-excalidraw-plugin#685]
Interesting observation, if I wrap an image into an Excalidraw darwing then the image appears in a markdown embed. If I have just the image, it doesn't.

**Idea 1**
I could create an excalidraw-img code block, and during the processing of the markdown document I could first replace the image link with the codeblock, so Excalidraw would process the image similar to how it does with drawings... thus have a workaround for displaying images in markdown embeds.

**Idea 2**
I think the reason for the difference is likely that in Excalidraw the image is embedded in base64 url while Obsidian includes the Obsidian url... I can post process the HTML to replace image urls with base64 copies.

**Comment by zsviczian** [comment]
I run the following code
```typescript
const mdDiv = createDiv();
const mdString = "![[Monkey.png]]";
const path = "/";
await MarkdownRenderer.renderMarkdown(mdString, mdDiv, path, myPlugin);
```
The result is:
```html
<div><p><span alt="Monkey.png" src="Monkey.png" class="internal-embed">Monkey.png</span></p></div>
```

I could process the resulting HTML and convert internal-embeds to IMG elements.

**Comment by zsviczian** [comment]
solved in 1.7.7
